encrypt.py

```diff
@@ -44,9 +44,7 @@ def keygen() -> bytes:
     type=int,
     help="Ritual ID obtained from a side channel",
 )
-@click.option(
-    "--coordinator-provider-uri", type=str, help="URI of the coordinator provider", required=True
-)
+@click.option("--coordinator-provider-uri", type=str, help="URI of the coordinator provider", required=True)
```

Comment:
Just formatting changes stemming from increasing max-line-length to 160 characters